src/RowOption/__tests__/index.test.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import * as React from 'react';
+import { render, fireEvent } from 'react-native-testing-library';
+
+import RowOption from '../RowOptionContainer';
+
+import { Touchable } from '../../Touchable';
+import { Icon } from '../../Icon';
+
+describe('RowOption', () => {
+  const onPress = jest.fn();
+
+  const { getByType, getAllByType, getByTestId } = render(
+    <RowOption
+      type="destination"
+      header="Very very long long long long long long long long"
+      subheader="10 km from center"
+      onItemPress={onPress}
+      onAddPress={onPress}
+      border="shaped"
+      info="Czech Republic"
+    />
+  );
+
+  it('should have city icon when type is destination', () => {
+    expect(getAllByType(Icon)[0].props.name).toBe('city');
+  });
+
+  it('should have dot separator once info prop is passed', () => {
+    expect(getByTestId('dot-separator')).toBeDefined();
+  });
+
+  it('should have triangle shape once border type is shaped', () => {
+    expect(getByTestId('triangle-shape')).toBeDefined();
+  });
+
+  it('should execute onItemPress function', () => {
```

Comment:
Can we please add test for `onAddPress`.
Also we just discussed that we should probably also test if event was actually fired like:
```
it('should execute onItemPress function', () => {
  fireEvent(getByType(Touchable), 'press');
  expect(onPress).toHaveBeenCalled(); 
});
```
( `onPress` in example will probably be renamed to `onItemPress` after we add extra test for `onAddPress`)

Reply:
Sure!